python/observation-publisher/tests/test_prediction_log_consumer.py

```diff
@@ -0,0 +1,194 @@
+from datetime import datetime
```

Comment:
I suppose the CI for the observation-publisher would come in later. 

In the meantime, can we start adding some basic dev docs (eg: running the tests locally), in a README file for this module?